packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -94,6 +96,16 @@
  */
 - (UIViewController *)createRootViewController;
 
+/**
+ * It assigns the rootView to the rootViewController
+ * By default, it assigns the rootView to the view property of the rootViewController
+ *
+ * @return: an instance of `UIViewController`
+ */
+- (UIViewController *)attachRootView:(UIViewController *) rootViewController
+                            rootView:(UIView *)rootView;
```

Comment:
The iOS convention is slightly different, what do feel about renaming the method like this:
```suggestion
 *   - (void)setRootView:(UIView *)rootView toRootViewController:(UIViewController *)rootViewController;
```
Also, it doesn't have to return anything, as in the implementation, we are not using the returned value.